packages/react-instantsearch/src/connectors/connectHighlight.js

```diff
@@ -0,0 +1,45 @@
+import createConnector from '../core/createConnector';
+import parseAlgoliaHit from '../core/highlight';
+
+const config = {
+  highlightPreTag: '<ais-highlight>',
```

Comment:
Can we add some randomness to pre and post tags? I know we did it here: autocomplete.js#132. This would provide something slighty maybe more secure :copyright: security selfmade consultant